test/common/setInterval.spec.ts

```diff
@@ -1,6 +1,12 @@
 'use strict';
 import {zoneSymbol} from "../../lib/common/utils";
 
+var isNode = true;
```

Comment:
I think this should do the trick:

1) `var isNode = typeof global == 'object';`
2) We should import `isNode` from `../../lib/common/utils`